prow/gerrit/client/client.go

```diff
@@ -684,3 +685,21 @@ func (h *gerritInstanceHandler) QueryChangesForProject(log logrus.FieldLogger, p
 		}
 	}
 }
+
+// ChangedFilesProvider lists (in lexicographic order) the files changed as part of a Gerrit patchset.
+// It includes the original paths of renamed files.
+func ChangedFilesProvider(changeInfo *ChangeInfo) config.ChangedFilesProvider {
```

Comment:
Let me know if you'd prefer this helper function lived in the source.go file instead. Wasn't sure where it made the most sense.

Reply:
`source` package is so far very lightweight functions that's safe to be imported by any package, such as `config`. I'm not seeing this function being that widely used, so let it be here should be fine